lambdas/data-migration2/package.json

```diff
@@ -39,6 +39,7 @@
     "pg": "^8.3.0"
   },
   "devDependencies": {
-    "@cumulus/test-data": "9.3.0"
+    "@cumulus/test-data": "9.3.0",
+    "@cumulus/message": "9.3.0"
```

Comment:
We may not need this import since the factory code handles the integration with cumulus/message, so we don't actually use this package here

Reply:
https://github.com/nasa/cumulus/pull/2390/commits/c1426a9975c50a4d15b75941a87e91e5cf4f9a7e